apps/site/lib/site_web/controllers/schedule/line_controller.ex

```diff
@@ -79,7 +83,8 @@ defmodule SiteWeb.ScheduleController.LineController do
         line_diagram:
           Enum.map(conn.assigns.all_stops, fn stop ->
             update_route_stop_data(stop, conn.assigns.alerts, conn.assigns.date)
-          end)
+          end),
+        today: conn.assigns.date_time |> DateTime.to_date() |> Date.to_iso8601()
```

Comment:
I think this could just be `conn.assigns.date |> Date.to_iso8601` rather than getting the `date_time` and converting that to a date (unless it's important that it be the "real" current date and not the service date?).

Reply:
IIRC, `date_time != date` here, so must use `date_time`

Reply:
Right, that's why I asked whether it was important that this be the current calendar date, or the current service date. `assigns.date_time |> to_date` will get you the current calendar date, `assigns.date` will get you the current service date.